mobile/src/components/common/button/styles.ts

```diff
@@ -32,6 +32,12 @@ const styles = StyleSheet.create({
   [`button${ButtonVariant.SECONDARY}Label`]: {
     color: AppColor.BRAND.BLUE_100,
   },
+  [`button${ButtonVariant.GREY}`]: {
```

Comment:
I think we will use this button only for canceletion
```suggestion
  [`button${ButtonVariant.CANCEL}`]: {
```